examples/jaxrs-json-provider-jettison/pom.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>org.superbiz</groupId>
+  <artifactId>jaxrs-json-provider-jettison</artifactId>
+  <version>8.0.0-SNAPSHOT</version>
+  <packaging>war</packaging>
+  <name>OpenEJB :: Examples :: JAXRS JSON Provider with Jettison </name>
+
+
+  <properties>
+    <version.javaee-api>8.0</version.javaee-api>
```

Comment:
Shouldn't we use `8.0-1` here? The current project-version is used for the TomEE implementation, which uses `8.0-1` atm?

Reply:
Interesting catch @rzo1 
I see all the pom.xml files from the examples/ module uses `8.0`.
I would prefer to open a new JIRA, PR and email thread to discuss this change since it affects +176 examples.

Reply:
I'm fine with this :)

Reply:
Let's do that yes.

Reply:
Follow up email thread for the update javaee-api to 8.0-1: https://markmail.org/message/7asxv6az7s5q56yl